examples/example_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I would encode just the "geo" metadata because this could confuse the reader about the internal structure of metadata.

The real structure is `b"geo": "{<geometadata>}"` instead of `"geo": {<geometadata>}`. One option is to store this:

```
# example_geometadata.json
{
  "columns": {},
  "primary_column": "",
  ...
}
```

Reply:
My thinking is it would be better to include both, because we could have an example where we show that the user can include both `geo` metadata _and_ their own custom metadata. That answers a potential question: "I'm already using some custom metadata in my Parquet files; does that conflict with adding GeoParquet metadata?"

It's true that the metadata is stored as bytes and not as strings, but for the informative purposes of this metadata, I don't think omitting the fact that `'geo'` is of type `bytes` rather than `str` is an issue.

Reply:
OK, I think we can add later documentation clarifying the format of the JSON file, and refer to the spec for the metadata encoding.

Reply:
Note that the `b"geo"` vs `"geo"` is only a pyarrow interface issue (which exposes metadata as bytes), the actual Parquet metadata key/values are stores as UTF8 strings, which is mentioned in the spec:

https://github.com/opengeospatial/geoparquet/blob/d8592c1d540bb9f9b3a4a4f41d6b0dc4978736e2/format-specs/geoparquet.md?plain=1#L24

(this was clarified in https://github.com/opengeospatial/geoparquet/pull/6)